docs/topics/multiplatform/native-cocoapods.md

```diff
@@ -185,7 +185,8 @@ If you already have a project, you can add and configure the Kotlin CocoaPods Gr
                 // Specify the framework linking type. It's dynamic by default. 
                 isStatic = false
                 // Dependency export
-                export(project(":anotherKMMModule"))
+                // Uncomment if you have another module in your project:
```

Comment:
I'm guessing I also have to change `anotherKMPModule` to an actual name? Should we make a note of that here? Most often when docs ask me to uncomment stuff, I uncomment it and look no further :)